zoloto/marker.py

```diff
@@ -58,13 +54,12 @@ def spherical(self):
         x, y, z = self._tvec
         return Spherical(rot_x=arctan2(y, z), rot_y=arctan2(x, z), dist=self.distance)
 
-    @property
+    @cached_property
     def cartesian(self):
         return ThreeDCoordinates(*self._tvec)
 
-    @clru_cache(maxsize=None)
```

Comment:
Why aren't we caching the pose vectors any more?

Reply:
We are, manually. Notice the assignment at the bottom of the method. This now links the cache to the marker instance, and so is de-allocated when the marker drops out of scope